.buildkite/package.yml

```diff
@@ -5,6 +5,7 @@ notify:
     if: "build.state != 'passed'"
 
 env:
+  PLATFORMS: "+all linux/amd64 windows/amd64 darwin/amd64"
```

Comment:
I'm not sure if we support `darwin/amd64` or any other platforms

Reply:
See https://github.com/elastic/beats/blob/a156701b0302ca0e4e8676d2de96ebf2c1b7879d/dev-tools/mage/platforms.go/#L28